README.md

```diff
@@ -53,9 +53,38 @@ This default may be changed in a future breaking release. In the meantime, manua
   (binding [org.httpkit.client/*default-client* sni-client/default-client]
     <...>)
 ```
-
 See `org.httpkit.client/*default-client*` docstring for more details.
 
+## Using Unix Domain Sockets in client and server
+
+Java native Unix Domain Sockets are available since Java 17, so to maintain backwards compatibility with JVMs prior to this,
+resolving SocketAddresses and SocketChannels is pluggable in both client and server.
+
+### Client of a Unix Domain Socket
+
+To talk HTTP via a Unix Domain Socket at `/tmp/test.sock`, create your own implementations of `:address-finder` and `:channel-factory` in a customised client:
+
+```clojure
+(let [client (client/make-client {:address-finder (fn [uri]
+                                                    (UnixDomainSocketAddress/of "/tmp/test.sock"))
+                                  :channel-factory (fn [address]
+                                                      (SocketChannel/open StandardProtocolFamily/UNIX))})]
+   (client/get "http://foobar"))
```

Comment:
I may be missing something obvious here, but this example looks strange.

The client in `client/get` is referring to the `HttpClient` in the `let`.
Was the intention maybe to call `org.httpkit.client/get` with `{:client <client-in-let>}` opts?

Reply:
Checked at the REPL and made some suggested tweaks to the examples. Marking as resolved 👍